test/integration/generated/nameChecker/README.md

```diff
@@ -1,27 +1,87 @@
-## Azure SearchClient SDK for JavaScript
+# Azure SearchClient library for JavaScript
```

Comment:
_Not blocking for this PR, but we should address this later, so please log an issue_

Our guideline for readme title is slightly different...

This should be `Azure Search client library for JavaScript` i.e `Azure <Name of the service> client library for JavaScript`. Same for the next line.

Reply:
Done.